src/Pages/ExplorePage/ExplorePage.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom';
+
+import Card from '../../Components/Card/Card'
+import SearchBar from '../../Components/SearchBar/SearchBar'
+import { useVideo } from '../../Context/Video-Context';
+import { data } from '../../Data/Data'
+import './ExplorePage.css'
+
+function ExplorePage() {
```

Comment:
This is a great practice that all your pages are composed of different components and do not contain anything else